www/docs/customization/ko.md

```diff
@@ -0,0 +1,106 @@
+# Docker Images with Ko
+
+> Since v1.14.0.
+
+You can also use [ko][] to build and publish Docker container images.
+
+Please notice that Ko will build your binary again.
+That shouldn't increase the release times too much, as it'll use the same build
+options as the [build][] pipe when possible, so the results will probably be cached.
+
+!!! warning
+    Ko only runs on the publish phase, so it might be a bit hard to test — you
+    might need to push to a fake repository (or a fake tag) when playing around
+    with its configuration.
+
+```yaml
+# .goreleaser.yaml
+kos:
```

Comment:
Can this be `ko:` instead?

Reply:
it's pluralized because it can be multiple KO configurations... is the pluralization a problem? If it is, I can change...

Reply:
We could build an OCI image with ko for different binaries built from the same repository, which is why I designed it that way.